<launch>
	<node name="training_generator" pkg="planning_abstraction_net" type="training_data_generator" output="screen">
	</node>

	<node name="planner" pkg="planning_abstraction_net" type="planner" output="screen">
	</node>
</launch>
